pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -237,6 +237,7 @@ public void deleteSegmentFile() {
   private final String _timeColumnName;
   private final List<String> _invertedIndexColumns;
   private final List<String> _noDictionaryColumns;
+  private final List<String> _varLengthDictionaryColumns;
```

Comment:
Same here, why does RT need to know about var length dict columns if this PR is specifically for offline?